DESCRIPTION

```diff
@@ -0,0 +1,45 @@
+Package: bpmodels
+Title: Analysing transmission chain statistics using branching process
+    models
+Version: 0.3.0
+Authors@R: c(
+    person("Sebastian", "Funk", , "sebastian.funk@lshtm.ac.uk", role = "aut",
+           comment = c(ORCID = "https://orcid.org/0000-0002-2842-3406")),
+    person("Zhian N.", "Kamvar", , "zkamvar@gmail.com", role = "ctb",
+           comment = c(ORCID = "https://orcid.org/0000-0003-1458-7108")),
+    person("Flavio", "Finger", , "flavio.finger@epicentre.msf.org", role = "aut",
+           comment = c(ORCID = "https://orcid.org/0000-0002-8613-5170")),
+    person("James M.", "Azam", , "james.azam@lshtm.ac.uk", role = c("aut", "cre"),
+           comment = c(ORCID = "https://orcid.org/0000-0001-5782-7330"))
+  )
+Description: Provides methods to analyse and simulate the size and length
+    of branching processes with an arbitrary offspring distribution. These
+    can be used, for example, to analyse the distribution of chain sizes
+    or length of infectious disease outbreaks, as discussed in Farrington
+    et al. (2003) <doi:10.1093/biostatistics/4.2.279>.
+License: MIT + file LICENSE
+URL: https://github.com/epiverse-trace/bpmodels,
+    https://epiverse-trace.github.io/bpmodels/
+BugReports: https://github.com/epiverse-trace/bpmodels/issues
+Depends:
+    R (>= 3.6.0)
+Suggests:
+    bookdown,
+    dplyr,
+    epiparameter,
+    ggplot2,
+    knitr,
+    lubridate,
+    rmarkdown,
+    testthat,
+    truncdist
+VignetteBuilder:
+    knitr
+Remotes:
+    github::epiverse-trace/epiparameter
```

Comment:
I didn't go for a full review just yet but I want to flag early that you cannot use `Remotes` on CRAN. There are alternatives but they have their own issues. Your best bet is to:
- either wait that epiparameter is on CRAN
- or drop epiparameter from `Suggests`

Reply:
Thanks for flagging @Bisaloo. Do you intend to do a full review or is the second slot open?

Reply:
I'm happy to let anyone else do it so it's still open. Let me know if you don't find anyone else.

Reply:
I will. Thanks.

Reply:
I can volunteer, will self assign now

Reply:
> * or drop epiparameter from `Suggests`

I think this is the more practical solution. `epiparameter` is only used in a few places in the vignette and should be straightforward to change here ae24211d5708d4c14501feb2cf88b96703530338. I will edit the vignette to remove `epiparameter` and return to this commit when `epiparameter` is put on CRAN.